blueprints/controllers/ikea_e1812/ikea_e1812.yaml

```
@@ -1,40 +1,51 @@
 # Blueprint metadata
 blueprint:
   name: Controller - IKEA E1812 Shortcut button
-  description: |
-    Controller automation for executing any kind of action triggered by an IKEA E1812 Shortcut button. Allows to optionally loop an action on a button long press.
+  description: 'Controller automation for executing any kind of action triggered by
+    an IKEA E1812 Shortcut button. Allows to optionally loop an action on a button
+    long press.
+
```

```suggestion
  description: |
    Controller automation for executing any kind of action triggered by an IKEA E1812 Shortcut button. Allows to optionally loop an action on a button long press.
```

Keeping the current format here

---

This somehow came over from the test yaml you posted earlier. I did not change this on purpose :)